### ISSUE TITLE
[BUG] bar background color is always white

### ISSUE THREAD

**Issue by Netspider** [arkayenro/arkinventory#1498]
**What version number you are using?**
3.09.55-tbc
(works in 3.09.52-tbc)

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Burning Crusade / Live

**What language is the game client set to?**
german

**Describe the bug**
I cannot change the background color of my bars. After choosing a color I press [OK] and the color is reset to white.

**To Reproduce**
1. open bag
2. (bar background is a very dark, slightly transparent blue)
3. Menü -> Einstellungen (menu -> settings)
4. Optionen -> Leisten -> Style -> Hintergrund (Options -> Bars -> Style -> Background)
5. (rectangle right of arrow is bright blue, looks like #0000FF)
6. click on rectangle right of arrow
7. (bar background changes to bright blue #0000FF without any transparency)
8. (when I change the darkness and transparency sliders the little rectangle next to the arrow changes to the correct color, the bar background stays bright blue)
9. (when I change the color in the circle the bar background changes to white #FFFFFF immediately, the rectangle next to the arrow still shows the correct color)
10. press [OK]
11. (the rectangle next to the arrow changes to white)

_if I drag the darkness slider all the way to the bottom the bar background changes from pure white to pure black! Is the color value rounded up somewhere so it is either 0 or 1?_

**Screenshots**
![ark-1](https://user-images.githubusercontent.com/347282/129476247-90e3ba4c-243c-4277-9194-10101e1fd0d4.jpg)
![ark-2](https://user-images.githubusercontent.com/347282/129476248-baa4952a-a4a9-44de-b341-ff0272313f22.jpg)
![ark-3](https://user-images.githubusercontent.com/347282/129476249-7c250f4f-59f5-43de-b3e5-986d31183b86.jpg)
![ark-4](https://user-images.githubusercontent.com/347282/129476250-6816accc-cf6b-44e3-9d86-f04ee55efae0.jpg)
![ark-5](https://user-images.githubusercontent.com/347282/129476251-77dea075-2f19-4323-932d-f28f1148e0bb.jpg)

**Additional context**
deleted WTF folder and ArkInventory-* is the only addon. First login. no other changes to the profile.

**Comment by arkayenro** [comment]
yes, you are correct.  its a locale issue with the decimal separator being a comma.  which is probably the cause of some of the other weird issues people are having

when i format the number to a string its getting a period for the decimal, not a comma, so when its converted back to a number at the other end the decimal is stripped off (as its meant to be a thousands separator).

i just dont know why its doing it that way

**Comment by arkayenro** [comment]
try [alpha 7](https://www.curseforge.com/wow/addons/ark-inventory/files/3426167)

**Comment by Netspider** [comment]
yes, this works!

**Comment by arkayenro** [comment]
could you check its still ok for you in [alpha 10](https://www.curseforge.com/wow/addons/ark-inventory/files/3430371)

**Comment by Netspider** [comment]
still working with alpha 10
